api/topn_agg.md

```diff
@@ -0,0 +1,71 @@
+# topn_agg() <tag type="toolkit" content="Toolkit" /><tag type="experimental" content="Experimental" />
+Produces an aggregate that can be passed to the [`topn` function][topn] to
+calculate the `n` most-frequent values in a column.
+```sql
+topn_agg (
+    FIXME INTEGER,
+    value AnyElement
+) RETURNS TopnAggregate
+```
+
+<highlight type="warning">
+Experimental features could have bugs. They might not be backwards compatible,
+and could be removed in future releases. Use these features at your own risk,
+and do not use any experimental features in production.
+</highlight>
+
+## Required arguments
+
+|Name|Type|Description|
+|-|-|-|
+|`FIXME`|`INTEGER`|The target number of most-frequent values|
```

Comment:
Does `topn_agg` only store `n` values, or does it store more, in case it needs them for re-aggregations?

Reply:
It may store more than n; @WireBaron ?

Reply:
It tracks significantly more than N, but we'll only return an error if you pass a value greater than `n` to `topn` with a `topn_agg`, if that makes any sense.